### ISSUE TITLE
Compile error: The filename or extension is too long

### ISSUE THREAD

**Issue by rieniter** [godotengine/godot#36717]
**Godot version:
3.2.1 custom build
**OS/device including version:**
Windows 10
**Issue description:**
![2020-03-02_053738](https://user-images.githubusercontent.com/34160378/75635392-45ff9580-5c48-11ea-9bf4-2c53cbe1e1b1.png)

This happen when i'm using this command:
scons platform=android target=release android_arch=arm64v8

**Comment by LinuxUserGD** [comment]
I think you need to do a ./gradlew build instead of directly compiling for android through scons

**Comment by akien-mga** [comment]
> I think you need to do a ./gradlew build instead of directly compiling for android through scons

No: https://docs.godotengine.org/en/3.2/development/compiling/compiling_for_android.html#building-the-export-templates

**Comment by rieniter** [comment]
![2020-03-02_113053](https://user-images.githubusercontent.com/34160378/75645658-609e3280-5c79-11ea-9e18-8330aec4544b.png)
Same problem with armv7

**Comment by Calinou** [comment]
@rieniter What's the full path to the Godot source directory on your PC?

**Comment by rieniter** [comment]
> @rieniter What's the full path to the Godot source directory on your PC?

It's 
D:\Engine\Godot\Clone Godot github\godot 3.2\godot-3.2

I can compile for windows normally, but only android get this problem

**Comment by malbach** [comment]
@rieniter just a hinch, but what is the full path to android sdk ?

**Comment by rieniter** [comment]
> @rieniter just a hinch, but what is the full path to android sdk ?
> 
> [EDIT] It compiles fine here, Win10 x64 VS2017

My Android SDK, NDK and gradle path were set in same folder of C:\Android SDK

**Comment by malbach** [comment]
> My Android SDK, NDK and gradle path were set in same folder of C:\Android SDK
OK, try to remove any space in path, might help

**Comment by rieniter** [comment]
> > My Android SDK, NDK and gradle path were set in same folder of C:\Android SDK
> 
> OK, try to remove any space in path, might help

Thank you, i tried to move to another folder without space and also change sdk folder to android studio and it worked

**Comment by Calinou** [comment]
Is handling SDK folders with spaces in the path something we can solve on our end?
